packages/grafana-ui/src/components/Logs/LogRowContextProvider.tsx

```diff
@@ -1,9 +1,10 @@
-import { DataQueryResponse, DataQueryError } from '@grafana/ui';
-import { LogRowModel } from '@grafana/data';
 import { useState, useEffect } from 'react';
```

Comment:
Same with this components. It's very Explore specific, not sure it will ever be used in other context - it belongs to explore feature IMO

Reply:
Good points, but how do we reference this from LogRows which belongs to GrafanaUI? Any suggestions @dprokop @torkelo

Reply:
That's tricky indeed. Let's leave it in grafana-ui for now and we will try to figure out a pattern for this behaviour later.